src/AppInstallerCLIE2ETests/InstallCommand.cs

```diff
@@ -193,5 +272,75 @@ private bool VerifyTestMsixInstalledAndCleanup()
 
             return TestCommon.RemoveMsix(InstallTestMsixName);
         }
+
+        private bool VerifyTestPortableInstalledAndCleanup(
+            string installDir,
+            string packageId,
+            string expectedCommandAlias,
+            string expectedFileName,
+            string expectedProductCode)
+        {
+            string installPackageRoot = Path.Combine(installDir, packageId);
+            if (!File.Exists(Path.Combine(installPackageRoot, expectedFileName)))
+            {
+                return false;

```

Comment:
Changed it so that cleanup still occurs even if first check fails, and also added a TODO noting that this will be replaced with uninstall when it is implemented.